### ISSUE TITLE
Wrong documentation on UseTestDataFrom

### ISSUE THREAD

**Issue by cuneytcalishkan** [serenity-bdd/serenity-core#2501]
Hello,

The Javadoc for net.thucydides.junit.annotations.UseTestDataFrom seems to be a copy/paste from net.thucydides.junit.annotations.Concurrent.

It states
```
Indicates that data-driven tests can and should be run in parallel. You can optionally indicate the number of threads to use. If not specified, a sensible value based on the number of processors will be used.
```

Best regards,
Dzhuneyt

**Comment by wakaleo** [comment]
Indeed. Could you propose a PR?

**Comment by cuneytcalishkan** [comment]
Sure, I will try to do that this weekend

**Comment by cuneytcalishkan** [comment]
Hello @wakaleo, here is a PR to fix the documentation https://github.com/serenity-bdd/serenity-core/pull/2508

**Comment by cuneytcalishkan** [comment]
Fixed in PR #2508
